modules/doc/content/newsletter/2024/2024_01.md

```diff
@@ -50,3 +76,8 @@ for a complete description of all MOOSE changes.
 ## PETSc-level Changes
 
 ## Bug Fixes and Minor Enhancements
+
+- Shorthand functor material property output was added using the [!param](/FunctorMaterials/GenericFunctorMaterial/output_properties) parameter. This automatically creates a [FunctorAux.md] and a constant monomial variable for output, which had to be done manually before.
+- Extra element IDs can now be used to assign elements to subdomains in the [ParsedSubdomainMeshGenerator.md].
+- The [PropertyReadFile.md] user object can now read multiple files, once every time it is executed. The execution is controlled with the [!param](/UserObjects/PropertyReadFile/execute_on) parameter.
+- The [NEML2](syntax/NEML2/index.md) constitutive model library was added to the tensor mechanics module as an optional dependency. NEML2 is built on top of libTorch and supports teh implementation of devide independent material models (with GPU support).
```

Comment:
```suggestion
- The [NEML2](syntax/NEML2/index.md) constitutive model library was added to the tensor mechanics module as an optional dependency. NEML2 is built on top of libTorch and supports the implementation of devide independent material models (with GPU support).
```

@dschwen One more typo here I didn't know how to fix - what did you mean insteade of "devide"?